src/client/Microsoft.Identity.Client/Region/RegionSource.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.;
+
+namespace Microsoft.Identity.Client.Region
+{
+    internal class RegionSource
```

Comment:
Isn't it better to send numbers instead of strings? Those headers can get pretty large.